frontend/public/components/resource-quota.jsx

```diff
@@ -57,23 +109,52 @@ export const getQuotaResourceTypes = (quota) => {
   return _.keys(specHard).sort();
 };
 
-const getResourceUsage = (quota, resourceType) => {
+export const getResourceUsage = (quota, resourceType, namespace, clusterOnly = false) => {
+  const reference = referenceFor(quota);
+  const isACRQ = reference === appliedClusterQuotaReference;
   const isCluster = isClusterQuota(quota);
   const statusPath = isCluster ? ['status', 'total', 'hard'] : ['status', 'hard'];
   const specPath = isCluster ? ['spec', 'quota', 'hard'] : ['spec', 'hard'];
   const usedPath = isCluster ? ['status', 'total', 'used'] : ['status', 'used'];
+  let totalUsed;
+  let used = _.get(quota, [...usedPath, resourceType]);
+
+  // ACRQ is a cluster-scoped resource that behaves like a namespaced resource
+  // It has cluster-scoped data as well as namespaced data
+  // "used" is the data for the current namespace and "totalUsed" is the cluster-scoped data
+  if (isACRQ) {
+    if (!clusterOnly) {
+      const allNamespaceData = _.get(quota, ['status', 'namespaces']);
+      if (namespace !== undefined) {
```

Comment:
If there isn't a specific reason we need to differentiate between `undefined` and `''` or `null`, use

```suggestion
      if (namespace) {
```